suites/krbd/rbd/tasks/rbd_fio.yaml

```diff
@@ -3,7 +3,7 @@ tasks:
 - ceph: null
 - rbd_fio:
     client.0:
-       fio-version: 2.2.9
+       fio-version: 2.2.11
```

Comment:
I am not saying remove the ability to override -- I am just saying these current "overrides" are redundant and should be removed.